dist/21-526EZ-schema.json

```diff
@@ -2,7 +2,458 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "title": "SUPPLEMENTAL CLAIM FOR COMPENSATION (21-526EZ)",
   "type": "object",
-  "definitions": {},
-  "properties": {},
+  "definitions": {
+    "email": {
```

Comment:
There's a built in email format for json schema, do you know if you need something different than that?

Reply:
Oh, I didn't know that. I copied this from `10-10EZ`, I think.